src/classes/Recipe.js

```diff
@@ -0,0 +1,49 @@
+import ingredientsData from '../data/ingredients'
+import Ingredient from './Ingredient'
+
+class Recipe {
+  constructor(recipeData) {
+    this.id = recipeData.id
+    this.image = recipeData.image
+    this.recipeIngredients = recipeData.ingredients
+    this.instructions = recipeData.instructions
+    this.name = recipeData.name
+    this.tags = recipeData.tags
+    this.ingredients = this.getAllIngredientsData()
```

Comment:
@jwasmer the more I think about this the more I think we need to make a test for it